### DISCUSSION TITLE
part-ii/2018-98/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#99] [Question comments]
Part II 2018 Number Fields: Paper 1, Section II, G 

https://questions.tripos.org/part-ii/2018-98/

**Comment by artie2000** [comment]
<details>
<summary>General approach:</summary>

Try to factor an ideal of norm $n^2+n+m$.

</details>

**Comment by dropdbassett** [comment]
<details><summary>Part (a)</summary>

Note the minimal polynomial of $\alpha = \frac{-1 + \sqrt{-p}}{2}$ is $T^2 + T + m$, and since $-p \equiv 1 \pmod 4$, we have $\mathcal{O}_L = \mathbb{Z}[\alpha]$.

Now if $q$ is a prime factor of $n^2 + n + m$, then $n^2 + n + m$ has a linear factor (mod $q$), and hence $(q)$ splits into two prime ideals of norm $q$.

If $n^2 + n + m < m^2$, we can pick $q$ such that $q < m$, and a bounding argument shows that the only elements of norm less than $m$ are integers (whose norms are thus square numbers, and can't be $q$). So these ideals are not principal, and thus the class group is non-trivial.
</details>

<details><summary>Part (b)</summary>

Using Dedekind on $\alpha$ as in part (a), along with the Minkowski bound, immediately shows that the class group of $\mathbb{Q}(\sqrt{-163})$ is trivial.
</details>
